cellprofiler/gui/cpframe.py

```diff
@@ -972,113 +972,119 @@ def __on_help_path_list(self, event):
-NumPy Developers; openGL © 2013 Khronos Group; Pillow © 1990-2013
-Python Software Foundation; progressbar © 2005 Nilton Volpato; SciPy ©
-2013 SciPy Developers; sklearn © 2007-2013 Scikit-Learn Developers;
-and zlib © 1995-2013 Jean-loup Gailly, Mark Adler under BSD licenses;
-(b) altgraph © 2004 Istvan Albert, Bob Ippolito and © 2010-2013 Ronald
-Oussoren; macholib © 2010-2013 Bob Ippolito, Ronald Oussoren;
-modulegraph © 2010-2013 Bob Ippolito, Ronald Oussoren; py2app ©
-2004-2006 Bob Ippolito, © 2010-2011 Ronald Oussoren; and vigra ©
-1998-2013 by Ullrich Koethe under MIT licenses; (c) libjpeg ©
-1991-1998 Thomas G. Lane under a license from the Independent JPEG
-Group found at https://github.com/aseprite/aseprite/blob/master/docs/
-licenses/libjpeg-LICENSE.txt; (d) matplotlib © 2003-2012 John Hunter,
-© 2012-2013 Michael Droettboom et al. under licenses found at
-http://matplotlib.org/users/license.html#copyright-policy; (e) Java
-Runtime Environment © 2013 Oracle, Inc. under the Oracle Binary Code
-License Agreement for the Java SE Platform Products and JavaFX found
-at http://www.oracle.com/technetwork/java/javase/terms/license/
-index.html; (f) Bioformats © 2000-2013 University of Dundee & Open
-Microscopy Environment; ice © 2013 ZeroC, Inc.; mySQLdb © 2013 Andy
-Dustman; omero © 2002-2013 Open Microscopy Environment; and ZeroMQ ©
-2007-2012 iMatix Corporation and Contributors; (g) wxWidgets ©
-1999-2005 Julian Smart, et al.; and wxPython © 1992-2006 Julian Smart,
-et al. and © 1996 Artificial Intelligence Applications Institute under
-the wxWindows license; (h) Intel Visual Fortran Compiler Run-time
-Library, Intel Fortran Portability Library, Intel OMP Runtime Library,
-Math Library for Intel Compilers, Short Vector Math Library for Intel
-Compilers under the Single User license provision for
-Resdistributables © 2013 Intel Corporation under the End User License
-Agreement for the Intel Software Development Products; and
-(i) FastEMD © Copyright (c) 2009-2012, Ofir Pele. under BSD licenses
-
-"""
-
-        icon_copyrights = get_icon_copyrights()
-        if additional_copyrights is not None:
-            additional_copyrights += icon_copyrights
-        button_list.append(("Additional licenses and attributions", 
-                            additional_copyrights))
-
-        for button_text, license_text in button_list:
-            sub_sizer = wx.BoxSizer(wx.HORIZONTAL)
-            sizer.Add(sub_sizer, 0, wx.EXPAND | wx.ALL, 5)
-            license_button = wx.Button(dlg, -1, button_text + " >>")
-            sub_sizer.Add(license_button, 0, wx.ALIGN_LEFT | wx.RIGHT, 3)
-            sub_sizer.Add(wx.StaticLine(dlg), 0, wx.ALIGN_CENTER_VERTICAL | wx.EXPAND)
-
+        with wx.Dialog(self, size=(640, 800)) as dlg:
+            dlg.Title = "About CellProfiler"
+            sizer = wx.BoxSizer(wx.VERTICAL)
+            dlg.SetSizer(sizer)
             panel = wx.lib.scrolledpanel.ScrolledPanel(dlg)
```

Comment:
The diff is kind of screwed up here. This is a scrolling panel for the whole dialog which solves the problem of fitting too much stuff onto a small screen.